src/lib/markbind/src/parser.js

```diff
@@ -263,7 +263,9 @@ class Parser {
 
     if ((/^h[1-6]$/).test(node.name) && !node.attribs.id) {
       const textContent = utils.getTextContent(node);
-      const slugifiedHeading = slugify(textContent, { decamelize: false });
+      let cleanedContent = textContent.replace(/&lt;/g, '');
+      cleanedContent = cleanedContent.replace(/&gt;/g, '');
```

Comment:
>Yup, I've now changed it to use the previously suggested fix.

👍 

Why not combine it into one though `&lt;|&gt;` :o
A comment here explaining why we clean `&lt;&gt;` in particular might be nice too